### ISSUE TITLE
Discussion on URL endings

### ISSUE THREAD

**Issue by carlinmack** [WDscholia/scholia#1498]
| URL                                                   | Status         |
|-------------------------------------------------------|----------------|
| https://scholia.toolforge.org/author                  | Redirects to / |
| https://scholia.toolforge.org/author/                 | ✔️              |
| https://scholia.toolforge.org/about                   | ✔️              |
| https://scholia.toolforge.org/about/                  | 404            |
| https://scholia.toolforge.org/author/Q97270           | ✔️              |
| https://scholia.toolforge.org/author/Q97270/          | 404            |
| https://scholia.toolforge.org/author/Q97270/curation  | ✔️              |
| https://scholia.toolforge.org/author/Q97270/curation/ | 404            |

As we can see, 

- [ ] there is inconsistent behaviour, where most pages have no ending /
- [ ] there are 404s instead of silent redirects

Thoughts?

**Comment by egonw** [comment]
Thoughts:

* `https://scholia.toolforge.org/author` is short for `https://scholia.toolforge.org/author.html` (for individual pages)
* `https://scholia.toolforge.org/author/` is short for `https://scholia.toolforge.org/author/index.html` (for category pages)

Following this pattern, all makes sense to me, considering an aspect landing page is like an category, and a page for a specific item (of some aspect) or actions (e.g. for curation or about). Yeah, I _do_ see your point :)

That leaves the question of the redirect on `author`...

**Comment by carlinmack** [comment]
Researching into this, the behaviour in the original post is described in the [Flask documentation](https://flask.palletsprojects.com/en/1.1.x/quickstart/#unique-urls-redirection-behavior). I see your logic with category vs item, however it seems nicer to use the `/`s as there is an in-built redirect?

I'll look into if it is possible to have the redirect's go the other way.

**Comment by carlinmack** [comment]
It seems we can disable `strict_slashes` but we would also need to set the canonical URL to without trailing slashes so that search engines don't see the pages as separate. https://stackoverflow.com/questions/33241050/trailing-slash-triggers-404-in-flask-path-rule

There doesn't seem to be a consistent best practise for trailing slashes (TS). For static sites, [TS are preferred ](https://discourse.gohugo.io/t/hugo-support-for-urls-without-a-trailing-slash/6763) as you don't need a webserver to serve extension-less web pages. On the other hand, GitHub, StackOverflow, Twitter, gov.uk etc. by default serve without TS and only gov.uk redirects to TS-less. 

In conclusion, it's unclear. I would probably lean towards disabling `strict_slashes` and having without TS as the canonical URL.

**Comment by egonw** [comment]
@fnielsen, your input is needed.
